### ISSUE TITLE
SA1028 (CodeMustNotContainTrailingWhitespace) does not properly handle multi-line documentation comments

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#821]
Currently the SA1028 analyzer does not report warnings inside a multi-line documentation comment, such as the following:

```csharp
/**
 * <summary>  
 * Summary.    
 * </summary>  
 */
```

In addition, the code fix for SA1028 treats `SyntaxKind.MultiLineDocumentationCommentTrivia` the same way it treats `SyntaxKind.MultiLineCommentTrivia`. This may be incorrect since these comments are documentation comments with internal structure similar to `SyntaxKind.SingleLineDocumentationCommentTrivia`.

**Comment by vweijsters** [comment]
Grabbing this
